### ISSUE TITLE
Wrong icon color for search inputs in b-theme.

### ISSUE THREAD

**Issue by andergrim** [jquery/jquery-mobile#6757]
Description:
The icon (looking glass) in the input type="search" element is black in the dark b-theme. The "clear text" icon that shows after a user begins to typ into the input turns out as expected (white). 

  Test page/Steps to reproduce/Outcomes:
http://jsbin.com/ofuhaw/980/edit 

  Platforms:
Latest stock Chrome browser for Android 4.3. Google Chrome browser v 31 for Ubuntu 13.04.

  Versions:
jQuery Mobile 1.4.0 RC1
jQuery 1.8.2

**Comment by jaspermdegroot** [comment]
Suggested solution: https://github.com/jquery/jquery-mobile/pull/7953#issuecomment-73098066

**Comment by jaspermdegroot** [comment]
We should add a demo to illustrate how you can change the classes of the search icon span with a ```data-``` attribute to leave out the ```ui-alt-icon``` class in case you use a dark theme.

**Comment by jaspermdegroot** [comment]
@arschmitz @gabrielschulhof 

I tried to add a demo for this but it doesn't work as expected.

Markup:
```
<form>
	 <label for="search-2">Search:</label>
	 <input type="search" name="search-2" id="search-2" data-theme="b" data-classes="{ 'ui-textinput-search-icon': 'ui-icon ui-icon-search' }">
</form>
```
Result:
The search icon only gets class ```ui-textinput-search-icon``` and the wrapper doesn't get the corner and shadow classes anymore.

Am I doing something wrong?

**Comment by arschmitz** [comment]
@jaspermdegroot yes remove the quotes around the object in your markup

**Comment by jaspermdegroot** [comment]
@arschmitz 

Removing the quotes around the object will result in invalid markup, but the problem was related to quotes. Need to use double quotes inside the object. This works:
```
data-classes='{ "ui-textinput-search-icon": "ui-icon ui-icon-search" }'
```

**Comment by arschmitz** [comment]
Fixed on 1.5
